spec/models/miq_worker/systemd_common_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe MiqWorker::SystemdCommon do
+  describe ".service_base_name" do
+    before { MiqWorkerType.seed }
+
+    it "every worker has a matching systemd target and service file" do
```

Comment:
Really want to make this a `providers_common` spec but we'll need a good way to determine which workers come from which engine.

This is how the container_common_spec is testing all worker deployment name lengths

Reply:
Do we?  It can't hurt to check all workers in all engines.  If someone changes a particular engine, then the probability they break their own engine is higher than breaking others, especially since we won't merge on red.